docs/application/native/guides/app-management/app-controls.md

```diff
@@ -808,6 +812,50 @@ The following common application controls are available:
 - [Voice Recorder](common-appcontrols.md#voice-recorder)
 - [VPN Service](common-appcontrols.md#vpn-service)
 
+<a name="URI"></a>
+## Application Control URI
```

Comment:
Global: 
Request you to follow sentence case for all headings: 
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization

Reply:
Done